docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+slackbot:
+  image: node:6
+  entrypoint: /app/entrypoint.sh
+  environment:
+    - DATABASE_URL=postgres://postgres:standup@database/standup
+    - PORT=3000
+    - LOG_LEVEL=10
+  expose:
+    - 3000
+  ports:
+    - 3000
+  working_dir: /app
+  volumes:
+    - ./:/app
+    - /app/node_modules
+  links:
+    - database:database
+database:
+  image: postgres:latest
+  environment:
+    - POSTGRES_DB=standup
+    - POSTGRES_PASSWORD=standup
+  expose:
```

Comment:
The postgres image has an `expose` directive, so I don't think it's necessary to do it here.